pkgs/by-name/gl/glance/package.nix

```diff
@@ -0,0 +1,24 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "glance";
+  version = "0.3.0";
+
+  src = fetchFromGitHub {
+    owner = "glanceapp";
+    repo = pname;
```

Comment:
`repo = pname;` is an anti-pattern as the pname or the repo should be able to change independent of each other.

Reply:
why not change than when/if it doesn't match? you would have to make a change to this if the repo name change anyway, so i don't get the justification for it being an anti-pattern.

Reply:
I don't think its an anti-pattern, but `buildXPackage (rec {` is. `buildGoPackage` doesn't support the new function passing style but if it did this would be more of an issue.

Reply:
@ehmry would doing it more like [this](https://github.com/NixOS/nixpkgs/blob/master/pkgs/by-name/re/reaction/package.nix) be preferred?